AWS_FLB_CHERRY_PICKS

```diff
@@ -48,3 +48,7 @@ https://github.com/PettitWesley/fluent-bit.git tail-modify-debug-context-immutab
 # S3 tag corruption fix
 https://github.com/PettitWesley/fluent-bit.git s3-str-fixes 38303131e049265277881c0d79935ad31fdd3e13
 
+# User friend message when storage.total_limit_size causes fs chunk deletion, before there was no indication
```

Comment:
these were already merged in master, I checked and ran build myself, it works

Reply:
Cool! Thanks, this is helpful